client/src/reducers/user.js

```diff
@@ -11,9 +11,15 @@ export default function UserReducer(state = { mentors: [], current: null }, acti
 			mentors: action.mentors
 		};
 	case 'SET_CURRENT_USER':
+		if (action.current === '') {
+			return {
+				...state,
+				current: null
+			};
+		}
 		return {
 			...state,
-			currentUser: action.currentUser
+			current: action.current
```

Comment:
`action.current === '' ? null : action.current`

This gets rid of the cumbersome `if`.